client/my-sites/themes/auto-loading-homepage-modal.scss

```diff
@@ -1,138 +1,65 @@
+@import "@automattic/typography/styles/fonts";
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 
-.themes__auto-loading-homepage-modal {
+.dialog__content.themes__auto-loading-homepage-modal {
 	position: relative;
-
-	h1 {
-		margin-bottom: 1em;
-		text-align: center;
-	}
-}
-
-.themes__auto-loading-homepage-modal-close-icon {
-	cursor: pointer;
-	width: 16px;
-	height: 16px;
-	position: absolute;
-	top: 10px;
-	right: 10px;
-
-	@include break-medium {
-		width: 24px;
-		height: 24px;
-		right: 20px;
-		top: 20px;
+	padding: 45px 45px 17px;
+	max-width: 640px;
+	box-sizing: border-box;
+	.auto-loading-homepage-modal__heading {
+		@extend .wp-brand-font;
+		font-size: $font-headline-small;
+		line-height: 1.1;
+		width: 100%;
+		margin-bottom: 15px;
 	}
 }
 
-.themes__auto-loading-homepage-modal-homepage-hint {
-	margin-left: 1.5em;
-	font-size: $font-body-small;
-	font-style: italic;
-	margin-top: -5px;
-}
-
 .themes__theme-preview-wrapper {
 	display: flex;
 	flex-direction: column;
-	align-items: center;
-	max-width: 700px;
 }
 
-.themes__autoloading-homepage-option-description {
-	font-size: $font-body-small;
-	min-height: 90px;
-
-	@include break-medium {
-		width: 70%;
-		min-height: 0;
-	}
+.themes__auto-loading-homepage-modal-close-icon {
+	cursor: pointer;
+	color: var(--color-text-subtle);
+	position: absolute;
+	top: 0;
+	right: 8px;
 }
 
-.themes__theme-preview-items {
-	display: flex;
-	flex-direction: column;
-	margin: 0 0 1em;
-	max-width: 700px;
-
-	@include break-medium {
-		flex-direction: row;
-	}
+.auto-loading-homepage-modal__description {
+	margin-bottom: 30px;
 }
 
-.themes__theme-preview-item {
-	img,
-	.themes__iframe-wrapper {
-		display: none;
-	}
-
-	@include break-medium {
-		border: 1px solid var(--color-border-subtle);
-		border-radius: 2px;
-		width: 48%;
-		margin-left: 2%;
-
-		img,
-		.themes__iframe-wrapper,
-		.themes__theme-preview-image-wrapper {
-			display: block;
-			height: 100%;
-		}
-
-		.themes__iframe-wrapper,
-		.themes__theme-preview-image-wrapper {
-			max-height: 235px;
-			overflow: hidden;
-		}
-
-		.form-label {
-			height: 100%;
-		}
-
-		.form-radio {
-			margin: 1em;
-		}
-
-		.form-radio__label {
-			margin: 1em 1em 1em 3em;
-		}
-
-		&:first-of-type {
-			margin-left: 0;
-			margin-right: 2%;
-		}
-	}
-
-	img {
-		width: 100%;
-		height: auto;
+.auto-loading-homepage-modal__checkbox {
+	margin-bottom: 52px;
+	.components-checkbox-control__label {
+		font-size: $font-body;
 	}
 }
 
-.themes__theme-preview-item-iframe-container {
+.auto-loading-homepage-modal__actions {
 	display: flex;
-	flex-direction: column;
-
-	.themes__iframe-wrapper {
-		flex-grow: 1;
-		overflow: hidden;
-		position: relative;
-		pointer-events: none;
-	}
-	iframe {
-		height: 357%;
-		width: 357%;
-		max-width: 357%;
-		transform: scale(0.28);
-		transform-origin: top left;
-		position: absolute;
-		top: 0;
-		left: 0;
+	justify-content: flex-end;
```

Comment:
Thanks for your comment!
I think aligning a button to the right is common for LTR languages. So I tried placing the button and the confirmation checkbox closer instead of moving the button to the left. https://github.com/Automattic/wp-calypso/pull/77881/commits/d172edec03c5da01ccb0a89b71db8e7ecf8cea1f

<img width="700" alt="Screen Shot 2023-06-12 at 15 26 28" src="https://github.com/Automattic/wp-calypso/assets/5287479/d3dae23b-e6bf-4cdf-bba1-c66323dc32b1">

What do you think? @miksansegundo CC: @lucasmendes-design @taipeicoder

Reply:
> What about moving the button to the left so it's closer to the checkbox? I think that would be a better UX. Not sure if aligning to the right side is a must-follow convention.

That's a good point, I've seen it both ways in Calypso, but let's go with `flex-end` for now. cc: @lucasmendes-design 

A couple of difference w.r.t. the Figma design:
![Screenshot 2023-06-12 at 3 29 14 PM](https://github.com/Automattic/wp-calypso/assets/797888/43df0cb7-1bcb-4eaa-8946-e6967a4055fa)

- `<p>` `margin-bottom` should be 32px.
- Modal content padding should be 48px 48px 16px 48px, following the rule of 8px system.
- The close button distance should be 16px 16px. In this case: `right:12px; top:4px;`
- The heading text should be 32px (2rem) instead of 36rem.

Reply:
Thank you for your comment, @taipeicoder!

>Modal content padding should be 48px 48px 16px 48px, following the rule of 8px system.
The close button distance should be 16px 16px. In this case: right:12px; top:4px;

Addressed this here 👍 https://github.com/Automattic/wp-calypso/pull/77881/commits/7f321bcaf28ba6d1a5727abe37b584a9e8c2af46

><p> margin-bottom should be 32px.

I did this intentionally since I wanted to place UI closer in cohesion. 
https://github.com/Automattic/wp-calypso/pull/77881#discussion_r1226147320

>The heading text should be 32px (2rem) instead of 36rem.

Addressed it here. https://github.com/Automattic/wp-calypso/pull/77881/commits/26cd7eb50252bbee88a968be1e96d4512c819d52
I also changed to use 32px for the `Unlock custom styles` modal.